servers/parser-graphql-wrapper/src/apollo/resolvers.ts

```diff
@@ -208,7 +211,11 @@ export const resolvers: Resolvers = {
     },
   },
   PocketMetadata: {
-    __resolveType() {
+    __resolveType(parent) {
+      // Note when a new type is added we need to add it here.
+      if ('htmlEmbed' in parent) {
```

Comment:
will need to figure out a better way to make this future proof, but this works for now.